swap_meet/item.py

```diff
@@ -1,2 +1,29 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0, age = None):
+        if not isinstance(age, int):
+            self.age = 0
+        self.id = uuid.uuid4().int if id is None else id
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+
+    def get_category(self):
+        return "Item"
+    
+    def condition_description(self):
```

Comment:
This works fine! It's a lot to read, though. How might we use a list or dictionary to organize our condition string values and have just one return statement?

Reply:
For example:
```python
conditions = [
    "heavily used",
    "acceptable",
    # so on and so forth
    
    # how do we return the right condition based on the data structure?
```